packages/core/src/Absolute/Absolute.spec.js

```diff
@@ -14,13 +15,17 @@ describe('Absolute', () => {
   })
 
   test('renders with flag and Child', () => {
+    const consoleError = console.error
```

Comment:
n-b but setting this up at the top of the spec, and using the `afterEach` hook to restore it is generally the best practice.

Reply:
I did it this way for this test because I didn't want to unintentionally suppress warnings coming from the other `Absolute` tests in the suite. This one test's warning was coming from the use of the `Flag` component.